src/test/java/seedu/recipe/model/recipe/RecipeDurationTest.java

```diff
@@ -1,98 +1,66 @@
 package seedu.recipe.model.recipe;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertFalse;
 import static org.junit.jupiter.api.Assertions.assertThrows;
-import static org.junit.jupiter.api.Assertions.assertTrue;
 
 import java.util.Objects;
 
 import org.junit.jupiter.api.Test;
 
-import seedu.recipe.model.recipe.exceptions.RecipeDurationInvalidArgumentLengthException;
-import seedu.recipe.model.recipe.exceptions.RecipeDurationInvalidDurationException;
 import seedu.recipe.model.recipe.unit.TimeUnit;
 
 public class RecipeDurationTest {
```

Comment:
Could add in multiple whitespace like "         " and null testing.

Reply:
for Model PR